### ISSUE TITLE
`authorizeEndpoint` url string is incorrect

### ISSUE THREAD

**Issue by Humandoodlebug** [MicrosoftDocs/msteams-docs#931]
In the second code block on the page, the `authorizeEndpoint` url is incorrect.
It is currently set to `"https://login.microsoftonline.com/" + context.tid + "/oauth2/authorize?" + toQueryString(queryParams)`
But should actually be `"https://login.microsoftonline.com/" + context.tid + "/oauth2/v2.0/authorize?" + toQueryString(queryParams)` according to the [Graph docs](https://docs.microsoft.com/en-us/graph/auth-v2-user) (I have found that the first link doesn't work in all circumstances, for example when retrieving calendar data from a user).

It also seems that the `resource` field in `queryParams` has been renamed `scope` though I haven't found any scenarios where this causes a problem right now.

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7d939a56-d1be-f2a2-3454-0f0c5144bc27
* Version Independent ID: ec4d5574-ddeb-9408-ad82-d86507a5111b
* Content: [Authentication for tabs using Azure Active Directory - Teams](https://docs.microsoft.com/en-gb/microsoftteams/platform/concepts/authentication/auth-tab-aad#feedback)
* Content Source: [msteams-platform/concepts/authentication/auth-tab-AAD.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/concepts/authentication/auth-tab-AAD.md)
* Product: **msteams**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by Gousia-MSFT** [comment]
@Humandoodlebug There are two different samples for Tab authentication.
1. [sample1](https://github.com/OfficeDev/microsoft-teams-sample-auth-node/blob/master/src/views/tab/simple/simple-start.hbs) uses resource field in queryParameters
![image](https://user-images.githubusercontent.com/45553518/64322050-6a532000-cfdf-11e9-89ad-75936ca2c852.png)
2. [Sample 2](https://github.com/OfficeDev/microsoft-teams-sample-auth-node/blob/master/src/views/tab/simple/simple-start-v2.hbs) uses Azure AD v2 auth and uses scopes in queryParameters.
![image](https://user-images.githubusercontent.com/45553518/64322237-ce75e400-cfdf-11e9-970d-b1d0aa4a24ae.png)

**Comment by Humandoodlebug** [comment]
@Gousia-MSFT to clarify, the problem is the sample on the docs page, which uses the v2 scope field format (`https://graph.microsoft.com/User.Read openid`), calls it resource (v1 naming) and uses the v1 endpoint (`"https://login.microsoftonline.com/" + context.tid + "/oauth2/authorize?" + toQueryString(queryParams)`).

The full sample, as it is currently written [here](https://docs.microsoft.com/en-gb/microsoftteams/platform/concepts/authentication/auth-tab-aad#navigate-to-the-authorization-page-from-your-popup-page) is this:
```javascript
microsoftTeams.getContext(function (context) {
    // Generate random state string and store it, so we can verify it in the callback
    let state = _guid(); // _guid() is a helper function in the sample
    localStorage.setItem("simple.state", state);
    localStorage.removeItem("simple.error");
    // Go to the Azure AD authorization endpoint
    let queryParams = {
        client_id: "YOUR_APP_ID_HERE",
        response_type: "id_token token",
        response_mode: "fragment",
        resource: "https://graph.microsoft.com/User.Read openid",
        redirect_uri: window.location.origin + "/tab-auth/simple-end",
        nonce: _guid(),
        state: state,
        // The context object is populated by Teams; the loginHint attribute
         // is used as hinting information
        login_hint: context.loginHint,
    };

    let authorizeEndpoint = "https://login.microsoftonline.com/" + context.tid + "/oauth2/authorize?" + toQueryString(queryParams);
    window.location.assign(authorizeEndpoint);
});
```
Here's the corrected version:
```javascript
microsoftTeams.getContext(function (context) {
    // Generate random state string and store it, so we can verify it in the callback
    let state = _guid(); // _guid() is a helper function in the sample
    localStorage.setItem("simple.state", state);
    localStorage.removeItem("simple.error");
    // Go to the Azure AD authorization endpoint
    let queryParams = {
        client_id: "YOUR_APP_ID_HERE",
        response_type: "id_token token",
        response_mode: "fragment",
        scope: "https://graph.microsoft.com/User.Read openid",
        redirect_uri: window.location.origin + "/tab-auth/simple-end",
        nonce: _guid(),
        state: state,
        // The context object is populated by Teams; the loginHint attribute
         // is used as hinting information
        login_hint: context.loginHint,
    };

    let authorizeEndpoint = "https://login.microsoftonline.com/" + context.tid + "/oauth2/v2.0/authorize?" + toQueryString(queryParams);
    window.location.assign(authorizeEndpoint);
});
```

Basically, [sample 2](https://github.com/OfficeDev/microsoft-teams-sample-auth-node/blob/master/src/views/tab/simple/simple-start-v2.hbs) is correct, but this isn't what's currently found [here](https://docs.microsoft.com/en-gb/microsoftteams/platform/concepts/authentication/auth-tab-aad#navigate-to-the-authorization-page-from-your-popup-page). It looks to me as if the sample in the docs (that second link) has been partially converted from v1 to v2, but left half way between in a nonfunctional state.

## TL;DR
Take [this code](https://github.com/OfficeDev/microsoft-teams-sample-auth-node/blob/master/src/views/tab/simple/simple-start-v2.hbs) and put it [here](https://docs.microsoft.com/en-gb/microsoftteams/platform/concepts/authentication/auth-tab-aad#navigate-to-the-authorization-page-from-your-popup-page). ;)
